src/assets/i18n/zh-hans.json

```
@@ -93,40 +93,40 @@
   "Review": "",
   "Save Selection": "",
   "Screenshots": "",
-  "Scroll to top": "",
+  "Scroll to top": "回到顶部",
   "Search Input Fields": "",
   "Section Help": "",
   "Select VDEV width": "",
   "Select a VDEV layout": "",
   "Select a disk size from inventory": "",
   "Selected": "",
   "Selected SSH connection uses non-root user. Would you like to use sudo with <i>/usr/sbin/zfs</i> commands? Passwordless sudo must be enabled on the remote system.\nIf not checked, <i>zfs allow</i> must be used to grant non-user permissions to perform ZFS tasks. Mounting ZFS filesystems by non-root still would not be possible due to Linux restrictions.": "",
-  "Set new password:": "",
-  "Sign In": "",
+  "Set new password:": "设置新密码",
+  "Sign In": "登录",
   "Similar Apps": "",
   "Subject Alternative Name": "",
   "Sudo Enabled": "",
   "The VM could not start because the current configuration could potentially require more RAM than is available on the system. Memory overcommitment allows multiple VMs to be launched when there is not enough free memory for configured RAM of all VMs. Use with caution. Would you like to overcommit memory?": "",
   "The {service} service has started.": "",
   "There are no records to show.": "",
   "Time in seconds after which current user session will be disconnected. Interacting with UI extends the session.": "",
-  "Token Lifetime": "",
-  "Token Settings": "",
-  "Token expired": "",
-  "Train: {train}": "",
+  "Token Lifetime": "Token生存时间",
+  "Token Settings": "Token设置",
+  "Token expired": "Token已过期",
+  "Train: {train}": "通道",
```

The `{train}` variable should be included in the translation

```suggestion
  "Train: {train}": "通道 {train}",
```

or

```suggestion
  "Train: {train}": "通道: {train}",
```